grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1321,20 +1351,20 @@ class QueryService implements Status {
         documents
     }
 
-     void updateOrspUserRoles (User user, ArrayList<String> newRoles) {
-         final session = sessionFactory.currentSession
-         final String query = ' insert into supplemental_role (version, role, user, user_id) values (:version, :role, :userName, :userId)'
-         final SQLQuery sqlQuery = session.createSQLQuery(query)
-         newRoles.each { it ->
-             sqlQuery.setLong("version", 0)
-             sqlQuery.setString("role", it)
-             sqlQuery.setString("userName", user.userName)
-             sqlQuery.setLong("userId", user.id)
-             sqlQuery.executeUpdate()
-         }
+    void updateOrspUserRoles(User user, ArrayList<String> newRoles) {
+        final session = sessionFactory.currentSession
+        final String query = ' insert into supplemental_role (version, role, user, user_id) values (:version, :role, :userName, :userId)'
+        final SQLQuery sqlQuery = session.createSQLQuery(query)
+        newRoles.each { it ->
+            sqlQuery.setLong("version", 0)
+            sqlQuery.setString("role", it)
+            sqlQuery.setString("userName", user.userName)
+            sqlQuery.setLong("userId", user.id)
+            sqlQuery.executeUpdate()
+        }
     }
 
-    void deleteOrspUserRoles (userId) {
+    void deleteOrspUserRoles(userId) {
```

Comment:
Revert all changes to this class - they are not deployment related. Feel free to put up a separate PR that addresses what you want these changes to accomplish.

Reply:
done